src/renderers/shared/stack/reconciler/__tests__/ReactErrorBoundaries-test.js

```diff
@@ -1158,8 +1164,11 @@ describe('ReactErrorBoundaries', () => {
       'Normal render',
       // BrokenComponentWillReceiveProps will abort rendering:
       'BrokenComponentWillReceiveProps componentWillReceiveProps [!]',
-      'ErrorBoundary unstable_handleError',
       ...(ReactDOMFeatureFlags.useFiber ? [
+        // Fiber unmounts an error boundary's children
+        'Normal componentWillUnmount',
+        'BrokenComponentWillReceiveProps componentWillUnmount',
```

Comment:
This one is also duplicated: see `BrokenComponentWillReceiveProps componentWillUnmount` later in the array.